test/modules/net/interpolate.py

```diff
@@ -35,3 +38,18 @@ def forward(self, x):
 
     def get_example_inputs(self):
         return (torch.randn(1, 2, 3, 4),)
+
+
+@tag.skip_if(
+    not HAS_TORCH_OVER_28_DEV,
+    reason="The case isn't supported yet. It will be supported from torch 2.8.0.dev",
+)
+class InterpolateOnePointFive(torch.nn.Module):
```

Comment:
Q. Why it's not supported in lower version?

A. 
- CASE(torch < 2.8.0.dev) 
  - it's converted to index. -> It's interpreted to "Multiple indices" case. -> it's not supported in tico yet.
- CASE(torch >= 2.8.0.dev)  
  - it's converted to upsample nearest 2d.  -> it converts directly to RNN(ResizeNearsestNeighbor) in tico. -> Can support 1.5 times.

Reply:
Can I ask what is "Multiple indices" case?

Reply:
@jinevening `IndexTensorVisitor` don't support multiple indices.

```py
        # TODO Support multiple indices
        if len(indices) - indices.count(None) > 1:  # type: ignore[arg-type]
            raise NotYetSupportedError(
                "Multiple indices is not supported yet in aten.index.Tensor"
            )
```

This is current implementaion. I left it as TODO.
I didn't look into what values were. :thinking: